app/views/requirements/index.html.erb

```diff
@@ -22,6 +25,9 @@
     <td>
       <%= requirement.within_months %>
     </td>
+    <td>
+      <%= link_to "delete", requirement, method: :delete, data: {confirm: "Really?"} %>
+    </td>
```

Comment:
This actually results in a 'post' request:
![image](https://cloud.githubusercontent.com/assets/140655/10758941/c4404200-7cb4-11e5-9e99-f48b33553e7e.png)

Do you think there is another way, which will result in an actual 'delete' request on `/requirements/[:id]`?